File: slack-reader/src/test/java/dev/number6/slackreader/generate/MessageGenerator.java
package dev.number6.slackreader.generate;

import dev.number6.slackreader.model.Message;
import uk.org.fyodor.generators.Generator;
import uk.org.fyodor.range.Range;

class MessageGenerator implements Generator<Message> {

    @Override
    public Message next() {
        return new Message("message", SlackReaderRDG.string(Range.closed(10, 500)).next());
    }
}

File: slack-reader/src/test/kotlin/dev/number6/slackreader/generate/MessageGenerator.kt
package dev.number6.slackreader.generate

import dev.number6.slack.model.Message
import uk.org.fyodor.generators.Generator
import uk.org.fyodor.generators.RDG
import uk.org.fyodor.range.Range

internal class MessageGenerator : Generator<Message> {
    override fun next(): Message {
        return Message("message", RDG.string(Range.closed(10, 500)).next())
    }
}